doc/install/plugin.rst

```diff
@@ -9,20 +9,20 @@ Linux install
 ~~~~~~~~~~~~~
 
 Obtain the Linux plugin from the link above. These instructions assume that you
-have installed Trelis in ``/opt/Trelis-16.0``. If you installed it somewhere
+have installed Cubit in ``/opt/Coreform-Cubit-2020.2``. If you installed it somewhere
 else, modify these instructions appropriately.
 
 These operations may need to be performed as the root user (sudo).
 
-1.  Unpack the tarball in ``/opt/Trelis-16.0/bin/plugins``.
-2.  Change directory to ``/opt/Trelis-16.0/bin/plugins/dagmc``.
+1.  Unpack the tarball in ``/opt/Coreform-Cubit-2020.2/bin/plugins``.
+2.  Change directory to ``/opt/Coreform-Cubit-2020.2/bin/plugins/dagmc``.
 3.  Run the install script: ``./install.sh``.
 
 OS/X install
 ~~~~~~~~~~~~
 
 Obtain the OS/X plugin from the link above. These instructions assume that you
-have installed Trelis in ``/Applications/Trelis-16.0``. If you installed it
+have installed Cubit in ``/Applications/Trelis-16.0``. If you installed it
```

Comment:
```suggestion
have installed Cubit in ``/Applications/Trelis-17.1.0.app``. If you installed it
```

this would depend on where the plugin come from... this is true with the last version I built on GitHub

Reply:
Have you built a version with Cubit 2020 or 2021?  I'd like to update the docs for one of those versions. (or does it still say Trelis in the directory name?)

Reply:
it has not been merged but it has been built:
https://github.com/svalinn/Trelis-plugin/actions/runs/702907904

Reply:
not test on 2021 versions yet

Reply:
so back to the original question: what is the path of the Cubit app for 2020?

Reply:
Sorry I missed this, I updated my suggestion...
`/Applications/Coreform-Cubit-2020.2.app` is the path for Cubit 2020.2